distribution/lib/Standard/Test/0.0.0-dev/src/Test.enso

```diff
@@ -207,3 +242,14 @@ type Behavior
        Checks if the behavior is a failure.
     is_fail : Boolean
     is_fail self = self.result.is_fail
+
+## PRIVATE
+
+type Clue
+    ## PRIVATE
+
+       Represents a clue as to why a test failed
+
+       Arguments:
+       - add_clue: either Nothing or a function which modifies a failure message
+    Clue_Data ~add_clue
```

Comment:
A newline here would be good.

Also why make the add_clue suspended? If it is a function-closure there is no point I think in suspending it as its not computed until given arguments _anyway_. In case of Nothing there is even less reason to do so.

```suggestion
    Clue_Data add_clue

```

Reply:
Fair enough, add_clue should always be a function